### ISSUE TITLE
Question: does this implement the wg-dynamic protocol?

### ISSUE THREAD

**Issue by aragilar** [foxcpp/wirebox#17]
It's unclear whether WGDCP refers to the protocol implemented by https://git.zx2c4.com/wg-dynamic/about/, or is a custom one (based on the usage of protobuf, I'm guessing it's a custom one). It might be helpful to mention that this is different to wg-dynamic.

**Comment by foxcpp** [comment]
Yeah, it is just my experiment that is different from wg-dynamic.
